app/docker/components/datatables/configs-datatable/configsDatatable.html

```diff
@@ -11,7 +11,7 @@
           ng-disabled="$ctrl.state.selectedItemCount === 0" ng-click="$ctrl.removeAction($ctrl.state.selectedItems)">
           <i class="fa fa-trash-alt space-right" aria-hidden="true"></i>Remove
         </button>
-        <button type="button" class="btn btn-sm btn-primary" ui-sref="docker.configs.new">
+        <button type="button" class="btn btn-sm btn-primary" ui-sref="docker.configs.new({id: ''})">
```

Comment:
By using a query parameter, we do not need to specify an empty id here. You can revert the changes to this file.